src/Controls/samples/Controls.Sample.Sandbox/SandboxShell.xaml.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+
+namespace Maui.Controls.Sample
+{
+	public partial class SandboxShell : Shell
+	{
+		public SandboxShell()
```

Comment:
Nobody worry. This is just adding the file for easier testing purposes. I haven't changed the sandbox to default to shell